app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job working on the devise setup so far but I think it would be great if you add `:confirmable` in User model to let the user confirm his email before logging in so kindly add it ✨⭐ as mentioned in the requirements ⭐💯✅


![image](https://github.com/ShaliniNayan/Blog-app/assets/58553711/8bc20407-c479-4cb1-a492-dd08e028a82e)